### PR TITLE
official assignment

### PR DESCRIPTION
**Pull request by pavolloffay**
This should be final version of master thesis assignment

### CODE REVIEW

**Comment by pavolloffay** [comment]
@Jiri-Kremser could you please review?

File: README.adoc [hunk 1]
```diff
@@ -52,36 +52,38 @@ Source and target are canonical paths.
...
-as concept drift and anomaly detection. Chosen algorithm will be implemented in Java language and module in case
-of good results will be merged into monitoring system Hawkular. This requires that module will be distributed
-under Apache Licence 2.0.
+==== Alert prediction in metric data based on time series analysis
+Student by application time series analysis will develop module which will
+predict incoming events based on incoming and historical time series data.
+This events are defined as alerts usually as conjunction or disjunction of
+simple clauses based on data from time series.
+
+Data has following structure <timestamp, value>. There are also available
```
Comment (Jiri-Kremser) [inline README.adoc]: Data >points<

File: README.adoc [hunk 2]
```diff
@@ -52,36 +52,38 @@ Source and target are canonical paths.
...
+metadata such as name, units, type of quantity defining if it’s monotonic
+function or random. These metric data are for instance result of measurement
```
Comment (Jiri-Kremser) [inline README.adoc]: monotonic or non-monotinic

File: README.adoc [hunk 3]
```diff
@@ -52,36 +52,38 @@ Source and target are canonical paths.
...
-defining if it’s monotonic function or random. These metric data are for instance result of measurement of free disk
-space, free memory, Java heap usage etc.
-
-Content of theoretical part should include comparison of time series data mining methods and deal with problems
-as concept drift and anomaly detection. Chosen algorithm will be implemented in Java language and module in case
-of good results will be merged into monitoring system Hawkular. This requires that module will be distributed
-under Apache Licence 2.0.
+==== Alert prediction in metric data based on time series analysis
+Student by application time series analysis will develop module which will
+predict incoming events based on incoming and historical time series data.
```
Comment (Jiri-Kremser) [inline README.adoc]: `s/incoming events based on incoming/future events based on continuously/`

File: README.adoc [hunk 4]
```diff
@@ -52,36 +52,38 @@ Source and target are canonical paths.
...
+that module will be distributed under Apache Licence 2.0.
 
 == Description in Czech
-==== Predikce udalosti v metrickych datech za pomoci metod strojoveho uceni
-Student za pomoci metod strojového učeni pro časové rady nebo za pomoci regresní
-analýzy vyvine modul, který na základe historických dat umožní včasnou detekci
-blížících se události. Tyto události jsou předem definovaný jako takzvaný
-výstrahy (alerts) většinou jako konjunkce či disjunkce jednoduchých podmínek
-vztahujících se k datum z časových řad.
+==== Predikce události v metrických datech za pomoci analýzi časových řad
```
Comment (Jiri-Kremser) [inline README.adoc]: analýzi -> analýzy

File: README.adoc [hunk 5]
```diff
@@ -52,36 +52,38 @@ Source and target are canonical paths.
...
+Student za pomoci analýzi časových řad vyvine modul, který na základě příchozích
+a historických dat umožní včasnou detekci blížících se události. Tyto události
```
Comment (Jiri-Kremser) [inline README.adoc]: události -> událostí

**Pull request merged**